packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraAndroidCameraxPlugin.java

```diff
@@ -145,6 +148,7 @@ public void onAttachedToActivity(@NonNull ActivityPluginBinding activityPluginBi
     systemServicesHostApiImpl.setActivity(activity);
     systemServicesHostApiImpl.setPermissionsRegistry(
         activityPluginBinding::addRequestPermissionsResultListener);
+    deviceOrientationManagerHostApiImpl.setActivity(activity);
```

Comment:
I think so after reading the docs so thanks for pointing this out! I don't want this change to get lost in this PR since it seems like a legitimate bug, so I filed https://github.com/flutter/flutter/issues/139937 and will follow up with a PR.